docs/using-controller-manager-with-kubeadm.md

```diff
@@ -53,7 +53,7 @@
 - Create a configmap containing the cloud configuration for cloud-controller-manager.
 
     ```shell
-    kubectl create configmap cloud-config --from-file=/etc/kubernetes/cloud-config -n kube-system
+    kubectl create secret generic cloud-config --from-file=/etc/kubernetes/cloud-config -n kube-system
```

Comment:
Instead of creating it here, I suggest to create a yaml file like CSI
https://github.com/kubernetes/cloud-provider-openstack/blob/master/manifests/cinder-csi-plugin/csi-secret-cinderplugin.yaml